docs/examples.md

```diff
@@ -0,0 +1,39 @@
+**Selecting the first match via a custom Swift matcher**
+
+When an application has exact duplicate elements (every attribute is the same,
+including their hierarchy), then the correct fix is to update the app to avoid
+duplicating the UI. When that's not possible, a workaround is to match on
+the first element.
```

Comment:
Instead of the first element, couldn't we generalize this to match on Nth element?
I believe that can be baked in to the API :)